src/desktop/modules/routes/auth.ts

```diff
@@ -138,6 +139,27 @@ export default (app, proxyURL) => {
         }
     }));
 
+  app.get('/user/v1/switch/:userId', async (req, res) => {
+    const userSDK = containerAPI.getUserSdkInstance();
+    if (!req.params.userId) {
+      logger.info({ msg: 'switch user rejected missing userID' });
+      res.status(400).send(Response.error("api.user.switch", 400, "failed to switch user", "BAD_REQUEST"));
+    }
+
+    try {
+      const sessionData = { userId: req.params.userId };
+      await userSDK.setUserSession(sessionData);
+      const result = {
+        response: "Success",
+        userSid: uuidv1()
```

Comment:
is the same sessionId is not stored in DB ?

Reply:
No, its not stored

Reply:
sessionId should be stored in DB and used in server side telemetry as well